src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -2,6 +2,8 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.http.util.Asserts;
+import org.bouncycastle.openpgp.PGPException;
+import org.bouncycastle.openpgp.PGPPublicKey;
```

Comment:
it would be great if this dependencies were not required by the services that use your custom `PgpEncryptionUtil` class.
Perhaps on a separate PR.